nemo/collections/asr/__init__.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # =============================================================================
+from . import models
```

Comment:
Do we need this line? If we do, let's change it to `import models`, would that be sufficient?

Reply:
changed